modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb

```diff
@@ -184,6 +184,7 @@ def special_issues_per_disability
           (form_attributes['disabilities'] || []).map { |disability| special_issues_for_disability(disability) }.compact
         end
 
+        # TODO: this does not seem to be working, specifically when there are secondary disabilities and not primary
```

Comment:
If we think there's a legit prob here that we should look into, we should prob remove this comment and put a ticket in the backlog to go investigate.